### PR TITLE
Custom label also get custom required and optional text

### PR DESCRIPTION
**Pull request by ajaxangular**
Please see my last commit, https://github.com/gcanti/tcomb-form-native/commit/e2a9beb2dee6fe43a9714b1def211046a25b1d4b
The first commit is not related to this PR.

Thx

### CODE REVIEW

**Comment by gcanti** [comment]
What if `this.getAuto()` returns `'none'` or `label` is not a string (e.g. a jsx)?

File: lib/templates/bootstrap/textbox.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 var React = require('react');
 var { View, Text, TextInput } = require('react-native');
+import TextField from 'react-native-md-textinput';
```
Comment (alvaromb) [inline lib/templates/bootstrap/textbox.js]: This is an external dependency but you haven't modified the `package.json`. Why exactly do you need the Material Design `TextInput`?

**Comment by alvaromb** [comment]
Hello @ajaxangular!

What is this PR for? Can you elaborate what features are you adding to the library?

File: lib/templates/bootstrap/textbox.js (hunk 1 above)
Comment (ajaxangular) [inline lib/templates/bootstrap/textbox.js]: actually I didn't mean to PR this commit. It's just for my project. My PR is related to next one, after this commit

Comment (alvaromb) [inline lib/templates/bootstrap/textbox.js]: You cant' submit both commits if one is not intended to enter in the PR. Please cherry-pick your commits or open another PR. And don't forget to tell us _why_ we should merge it and _what_ it actually solves.

**Comment by ajaxangular** [comment]
If I have option like this: 

`
const options = {

  i18n: {

    required: ' (required)',

    optional: ' (optional)'

  },

  fields: {

    FirstName: {

      label: 'My custom label'

    },
...
`
then My custom label with not have "(required)" or "(optional)" text after that custom label

**Comment by rmarquois** [comment]
Hello.

I've also the same issue with custom label and required text. It does not work.

Options look like this :

```
{
      i18n: {
        optional: '',
        required: ' *'
      },
      fields: {
        firstName: {
          label: I18n.t('subscriber-form-firstName')
        },
       ...
      }
}
```

Will a fix be published soon?

Thank you.

**Pull request closed without merging**